src/main/scala/org/antipathy/mvn_scalafmt/builder/ChangedFilesBuilder.scala

```diff
@@ -58,7 +58,7 @@ object ChangedFilesBuilder {
       else run(branch.substring(prefix.length))
 
     def processFunction(): Seq[File] = {
-      val diffOutput    = run(s"git diff --name-only --diff-filter=d $actualBranch")
+      val diffOutput    = run(s"git diff --name-only --diff-filter=d $actualBranch -- workingDirectory")
```

Comment:
I tested this command by making changes to the following files in this project:
```
.github/workflows/maven.yml
pom.xml
src/main/scala/org/antipathy/mvn_scalafmt/builder/LocalConfigBuilder.scala
```

When running the commands 
`git diff --name-only --diff-filter=d master` or `git diff --name-only --diff-filter=d master $(pwd)` 
all were correctly reported, however when running
`git diff --name-only --diff-filter=d master -- workingDirectory`
no results are returned. 

Could you confirm what the command here is actually doing?

Reply:
We should be checking for changes only in the working directory specified by the user (where main pom.xml file often exists), and Not the diff files in whole project.
I meant to use the "workingdirectory" variable passed to changed files function